source/frontend/src/features/mapSideBar/property/tabs/takes/update/models.ts

```diff
@@ -10,7 +10,7 @@ export const TakesYupSchema = Yup.object().shape({
   takes: Yup.array().of(
     Yup.object().shape({
       description: Yup.string().max(4000, 'Description must be at most ${max} characters'),
-      takeTypeCode: Yup.string().required('Take type is required'),
+      takeTypeCode: Yup.string().required('Take type is required').nullable(),
```

Comment:
hmm?

Reply:
This fixes an issue with the take validation where it was showing an error instead of the message